### DISCUSSION TITLE
I have question in HW2.

### DISCUSSION THREAD

**Discussion by cpark88** [seominjoon/kaist-ai605#41] [Q&A] [answered]
There was already same question, but I am still confused...;;; 
In the problem,

**_Transform your model into a token classification model by imposing  softmax  over the tokens instead of predefined classes. You will need to do this twice for each of start and end._**

Did you mean that we should use last token (or other tokens) to classify start or not (and end or not) among all tokens? If we have input sequence of 512, then should we identify which word is start (or end) among 512 position using last token embedding after last layer (or average embedding)?

**Comment by seominjoon** [comment, marked as answer]
You basically should classify each token into either 0 or 1 (for start). To do this, you first map each to token encoding (vector) to a scalar value, which indicates how likely each token is going to be the start. Then you have two options for translating it into probability distribution: you can perform sigmoid on each scalar value, or softmax over all scalar values of the tokens. What I meant is that you should do the latter.

**Reply by cpark88** [reply]
I understand!! Thanks Professor^^
